openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/StreamProcessor.java

```diff
@@ -442,9 +446,10 @@ public void stopRecording(Client c) {
 		});
 	}
 
-	void startConvertion(Recording rec) {
+	public boolean startConvertion(Recording rec) {
 		IRecordingConverter conv = rec.isInterview() ? interviewConverter : recordingConverter;
 		taskExecutor.execute(() -> conv.startConversion(rec));
+		return true;
```

Comment:
it seems method returns nothing but true
seems to be useless

Reply:
You can't mock it otherwise. Mockito won't let you mock the method if its returning void. But we need to mock this method, the problem is the method is in a @InjectedMock class. So you need to use the alternative syntax for Spy. but can only do that if method returns a value.